pkg/registry/registry.go

```diff
@@ -55,31 +57,31 @@ type Info struct {
 // New initializes KbrewRegistry, creates or clones default registry if not exists
 func New(configDir string) (*KbrewRegistry, error) {
 	r := &KbrewRegistry{
-		path: filepath.Join(configDir, registriesDirName),
+		path: filepath.Join(configDir),
 	}
 	return r, r.init()
 }
 
 // init creates config dir and clones default registry if not exists
 func (kr *KbrewRegistry) init() error {
-	// Generate registry path
-	if _, err := os.Stat(kr.path); os.IsNotExist(err) {
-		if err := os.MkdirAll(kr.path, os.ModePerm); err != nil {
-			return errors.Wrap(err, "failed to initialize kbrew registry")
-		}
-	}
+
+	home, err := homedir.Dir()
+	cobra.CheckErr(err)
```

Comment:
Does it panics out on error?

Reply:
It prints the error and terminates with exit status 1